app/forms/mappings/Formatters.scala

```diff
@@ -110,15 +110,14 @@ trait Formatters {
 
       private val baseFormatter = stringFormatter(nothingEnteredKey)
 
-
       override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], BigDecimal] =
         baseFormatter
           .bind(key, data)
           .right.map(_.replace(",", "").replace(" ", ""))
           .right.flatMap {
           case s if !s.matches(numericRegexp) =>
             Left(Seq(FormError(key, notANumberKey, args)))
-          case s if !s.matches(noDecimalsKey) && !s.matches(intRegexp) =>
+          case s if !s.matches(noDecimalsKey) && !s.matches(decimal2DPRegexp) =>
```

Comment:
`noDecimalsKey` this is a key, not a regex. 
Can you double check this with AFT-fe ? 
and regex that we should use is intRegexp

Reply:
Changed to below in next commit as discussed and to match with validation messaging:

```
case s if !s.matches(decimal2DPRegexp) =>
  Left(Seq(FormError(key, noDecimalsKey, args)))
```